ampcrowd/basecrowd/models.py

```diff
@@ -28,6 +28,13 @@ def __unicode__(self):
     class Meta:
         abstract = True
 
+    @staticmethod
+    def to_json(task_group):
```

Comment:
Good catch, the `json_view` decorator can automatically convert the dict so will do `to_dict`